src/main/java/org/mitre/synthea/helpers/SimpleCSV.java

```diff
@@ -37,9 +38,24 @@
 
     MappingIterator<LinkedHashMap<String, String>> it = mapper.readerFor(LinkedHashMap.class)
         .with(schema).readValues(csvData);
+    
 
     return it.readAll();
   }
+  
+  public static Iterator<LinkedHashMap<String, String>> parseLineByLine(String csvData) throws IOException {
```

Comment:
Please add a Javadoc comment for this new method. It should mostly be the same as `parse` method above but distinguish why someone would want to use this one instead. (ie, reduced memory usage).

Also please review the formatting on this method, it doesn't look like Checkstyle will approve of this